src/components/Block/TokenPicker.vue

```diff
@@ -22,6 +22,9 @@ const filteredAssets = computed(() =>
           .toLocaleLowerCase()
           .includes(props.searchValue.toLocaleLowerCase()) ||
         asset.contract_name
+          .toLocaleLowerCase()
+          .includes(props.searchValue.toLocaleLowerCase()) ||
+        asset.contract_address
```

Comment:
It's unlikely that anyone would type the address characater by character so we could limit it to an exact match (`asset.contract_address === props.searchValue`).

Reply:
Yes agree, it's better to do exact match here to avoid collision with others token symbol